RIE/RIE198.xml

```diff
@@ -25,7 +25,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                 <msDesc>
 
                     <msIdentifier>
-                        <repository cert="low">In situ</repository>
+                        <repository ref="LOC1310Aksum">In situ <!-- To be more specific, nowadays the inscription is in the Church of the Four Animals --></repository>
```

Comment:
this can be encoded better, instead of staying in a comment. please write Church of the Four Animals and if an Institution record does not exist we need to create one.

Reply:
Working on it.